internal/pkg/vm/actor/builtin/miner/miner.go

```diff
@@ -21,82 +22,10 @@ import (
 	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/exec"
 )
 
-var log = logging.Logger("miner-actor")
-
```

Comment:
The re-arrangement makes it difficult for reviewers to understand what has and hasn't changed. Please resist the urge to make cosmetic changes when performing an extensive functional change. Land the re-arrangement in a "prefactor" PR, or just leave it for another day.